packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt

```diff
@@ -247,7 +247,7 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
         view: LottieAnimationView
     ) {
         val color: Int = if (colorFilter.getType("color") == ReadableType.Map) {
-            ColorPropConverter.getColor(colorFilter.getMap("color"), view.context)
+            ColorPropConverter.getColor(colorFilter.getMap("color"), view.context) ?: 0
```

Comment:
0 is quite vague. using Colors is clear imo. Maybe Colors.Black?

```suggestion
            ColorPropConverter.getColor(colorFilter.getMap("color"), view.context) ?: Colors.Black
```

Reply:
or `Color.TRANSPARENT` if transparent is wanted. I just think using the `Color` api has clearer intent